src/App.tsx

```diff
@@ -1,222 +1,208 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import {
+  getTodos,
+  updateTodo,
+  createTodo,
+  deleteTodo,
+} from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo, TodoTitle, SelectedType } from './types/Todo';
+import { ErrorType } from './components/ErrorType/ErrorType';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [filter, setFilter] = useState<string>(SelectedType.ALL);
+  const [showAll, setShowAll] = useState(true);
+  const [isDeleting, setIsDeleting] = useState<number[]>([]);
+  const [isChangeAllTodos, setIsChangeAllTodos] = useState<number[]>([]);
+  const [isError, setIsError] = useState('');
```

Comment:
Prefix `is` is usually used in names for boolean values. It can be confusing for other developers to put it for other values